client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -185,16 +199,14 @@ class DomainDetailsForm extends PureComponent {
 		} );
 	}
 
+	getMainFieldValues() {
+		const mainFieldValues = Object.assign( {}, formState.getAllFieldValues( this.state.form ) );
+		mainFieldValues.phone = toIcannFormat( mainFieldValues.phone, countries[ this.state.phoneCountryCode ] );
+		return mainFieldValues;
+	}
+
 	getAllFieldValues() {
```

Comment:
we only call `this.getAllFieldValues()`  in one place, and it's so simple now that it's probably better to drop the function altogether